create_issues.rb

```diff
@@ -0,0 +1,58 @@
+module CreateIssues
```

Comment:
Please store all your helper methods in one module called for example HelperMethods